packages/devtools_app/lib/src/timeline/timeline_service.dart

```diff
@@ -9,9 +9,8 @@ import '../config_specific/allowed_error.dart';
 import '../globals.dart';
 import '../profiler/cpu_profile_service.dart';
 import '../vm_service_wrapper.dart';
-import 'timeline_controller.dart';
+import 'timeline_controller.dart' hide Timeline;
```

Comment:
having to hide "Timeline" seems odd given it is defined in this package. Can you give the classes unique names instead or remove this if it is spurious.

Reply:
renamed to TimelineBase so now this collision is avoided